lib/ClangImporter/ImportMacro.cpp

```diff
@@ -125,6 +125,11 @@ static ValueDecl *importNumericLiteral(ClangImporter::Implementation &Impl,
         return nullptr;
     }
 
+    auto &ctx = DC->getASTContext();
+    auto *constantTyNominal = constantType->getAnyNominal();
+    if (!constantTyNominal)
```

Comment:
Should we add an assert at the other call sites too then, to avoid propagating nulls?